modules/test/playwright/tests/questions-web/pages/QuestionsPage.ts

```diff
@@ -0,0 +1,51 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {Page} from '@playwright/test';
+
+import fillAndClickOutside from "../../../utils/fillAndClickOutside";
+
+export class QuestionsPage {
+	readonly page: Page;
+
+	constructor(page: Page) {
+		this.page = page;
+	}
+
+	async goto() {}
+
+	async addNewQuestion(
+		questionBody: string,
+		questionTitle: string,
+		tagName: string
+	) {
+		await this.goto();
+		await this.page
+			.locator('li')
```

Comment:
We can simplify this locator like this `this.page.getByRole('button', { name: 'Ask Question' })`  
 `.click();`